tests/integration/test_metrics.py

```diff
@@ -799,7 +799,7 @@ def test_transaction_metrics_count_per_root_project(
 
     event, _ = transactions_consumer.get_event()
 
-    metrics_by_project = metrics_by_name_group_by_project(metrics_consumer, timeout=2)
+    metrics_by_project = metrics_by_name_group_by_project(metrics_consumer, timeout=4)
```

Comment:
On my machine, 2 -> 3 already saw significant improvements, bumping it to 4 for further stability.